src/styles/fab.css

```diff
@@ -1,30 +1,38 @@
-.fab-btn{
-    position: fixed;
-    bottom: 80px;
-    right: 60px;
-    z-index: 999;
-    border: 1px solid white;
-    font-size: 24px;
-    width: 50px;
-    height: 50px;
-    border-radius: 50%;
+.fab-btn {
+  position: fixed;
+  background: #000000bd;
+  bottom: 38px;
+  right: 20px;
+  z-index: 999;
+  border: 1px solid white;
+  font-size: 24px;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
 }
 
-.fab-btn:hover span{
-    display: block;
-    animation: fab-animation 1.2s linear infinite;
+.fab-btn:hover span {
+  display: block;
+  animation: fab-animation 1.2s linear infinite;
 }
 
 .fab-btn:hover {
-    background-color: white;
-    color: black;
+  background-color: white;
+  color: black;
 }
 
 @keyframes fab-animation {
-    50% {
-        transform: translateY(-0.5rem);
-    }
-    100% {
-        transform: translateY(0rem);
-    }
-}
\ No newline at end of file
+  50% {
+    transform: translateY(-0.5rem);
+  }
+  100% {
+    transform: translateY(0rem);
+  }
+}
+
+@media (max-width: 600px) {
+  .fab-btn {
+    bottom: 38px;
```

Comment:
Both the viewports have same `bottom` and `right` values now

Reply:
I forgot to update the previous changes, apologies for the inconvenience